### PR TITLE
feat(Tooltip): add shouldFitContainer to Tooltip

### PR DESCRIPTION
**Pull request by JakeXu**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## [0.11.7]
 ### Added
+- Support `shouldFitContainer` in `Tooltip`
```
Comment (rayhan-ab) [inline CHANGELOG.md:9]: should be in 0.11.8 even if we don't bump the version in this commit because 0.11.7 is already released

**Review by rayhan-ab — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "justice-ui-library",
-  "version": "0.11.7",
+  "version": "0.11.8",
```
Comment (rayhan-ab) [inline package.json:3]: could we bump the version laterin this open [PR](https://github.com/AccelByte/justice-ui-library/pull/111)? both are small fixes, so i think no need to bump the version twice as we also already bump the version today and yet to merge it to AP

Comment (rayhan-ab) [reply package.json]: and yeah i think we can remove #patch from the commit message. wdyt?

Comment (JakeXu) [reply package.json]: okay, revert to 0.11.7

**Pull request merged**